build.gradle

```diff
@@ -162,7 +162,7 @@ dependencies {
     configurations.all {
         resolutionStrategy {
             force("com.google.guava:guava:32.1.3-jre") // CVE for 31.1
-            force("org.eclipse.platform:org.eclipse.core.runtime:3.29.0") // CVE for < 3.29.0
+            force("org.eclipse.platform:org.eclipse.core.runtime:3.30.0") // CVE for < 3.29.0
```

Comment:
`3.30` had CVE. @dbwiddis can you confirm?

Reply:
It was 3.26 with CVE.  3.29 and higher is fine.

Reply:
And Mend is happy: https://github.com/opensearch-project/flow-framework/pull/255/checks?check_run_id=19316680248